### PR TITLE
Buying the wizard hardsuit gives you the no-breathing superpower

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Wizards no longer have to get an oxygen tank!
You can call it a quality of life or terrible addition, but personally I think it's better with this
:cl:
 * tweak: Buying the wizard hardsuit removes your need to breathe.

### CODE REVIEW

**Comment by SNilWillIllShill** [comment]
This plus Firebird can allow for vault exploration, and maybe a wiz gimmick like murdering the Trader/Merchant vox and taking over their shop to sell your rare vault loot to the station.

**Comment by Arthurdentist** [comment]
Wouldn't it just be better to give him like, a Gem encrusted oxygen tank?

**Comment by Blithering** [comment]
You would need a mask as well, wizards spawn with empty internals boxes

**Comment by Arthurdentist** [comment]
I checked and wizards appear to spawn with normal internal boxes, airtank, mask and all. I know  they didn't use to but it seems they do now.

File: code/game/gamemodes/wizard/artifacts.dm [hunk 1]
```diff
@@ -58,6 +58,12 @@
 	/obj/item/clothing/suit/space/rig/wizard,\
 	/obj/item/clothing/head/helmet/space/rig/wizard)
 
+/datum/spellbook_artifact/scrying/purchased(mob/living/carbon/human/H)
```
Comment (Arthurdentist) [inline code/game/gamemodes/wizard/artifacts.dm]: I just noticed that this PR isn't giving wizards who buy the hardsuit no breath, its giving those who buy the scrying orb no breath

Comment (boy2mantwicethefam) [reply code/game/gamemodes/wizard/artifacts.dm]: Okay, this is epic

**Comment by boy2mantwicethefam** [comment]
I was thinking no-breathe because it feels awkward to choose between an oxygen tank and a staff, and the ability to not need to breathe is fitting cause of magic

**Comment by Arthurdentist** [comment]
wizard hardsuit can put an oxygen tank into the suit slot as far as I know, will check to test this

**Comment by boy2mantwicethefam** [comment]
I know, that's the point, a slot that is used usually for magical items like staves is instead used for an oxygen tank, and I felt that would be too awkward, hence giving no-breathing to wizards who buy the hardsuit

**Comment by KingofHONK** [comment]
cant you just put the 02 tank in a pocket slot?

**Pull request closed without merging**